### DISCUSSION TITLE
Welcome to coreui-free-bootstrap-admin-template Discussions!

### DISCUSSION THREAD

**Discussion by mrholek** [coreui/coreui-free-bootstrap-admin-template#580] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by sdancer75** [comment]
How to keep my header fixed at the top of the CSideBar? CSideBar always overlays part of the AppHeader
